package.json

```diff
@@ -30,8 +30,7 @@
 		"svelte": "^3.54.0"
 	},
 	"devDependencies": {
-		"@babeard/svelte-heroicons": "^2.0.0-rc.0",
-		"@rgossiaux/svelte-headlessui": "^1.0.2",
```

Comment:
These pkgs are abandoned. This breaks the Switch component, unfortunately. I'll address that component eventually